src/utils/Utils.ts

```diff
@@ -65,6 +65,28 @@ export default class Utils {
     });
   }
 
+  public static getEponymousAttributeValue(resourceInstance: ResourceInstance): string {
+    if (!resourceInstance.resourceType.eponymousAttribute) {
+      if (!resourceInstance.id) {
+        return '';
+      }
+      return resourceInstance.id;
+    }
+    const eponymousAttribute = resourceInstance.resourceType.attributes.find( (resourceTypeAttribute) => {
+      return (resourceTypeAttribute.id === resourceInstance.resourceType.eponymousAttribute);
+    });
+    if (!eponymousAttribute) {
+      return '';
+    }
+    const attribute = resourceInstance.attributes.find( (resourceInstanceAttributes) => {
+      return (resourceInstanceAttributes.name === eponymousAttribute.name);
+    });
+    if (!attribute) {
+      return '';
```

Comment:
If the search for the attribute was not successful, we should still try to return the id here.

Reply:
I refactored the whole function to reduce the number of if statements